packages/tasks/src/model-data.ts

```diff
@@ -78,7 +78,7 @@ export interface ModelData {
 	 */
 	widgetData?: WidgetExample[] | undefined;
 	/**
-	 * Parameters that will be used by the widget when calling Inference API
+	 * Parameters that will be used by the widget when calling Inference Endpoints (serverless)
```

Comment:
Sometimes we use `Inference Endpoints (serverless)` and sometimes `Serverless Inference Endpoints`. Let's try to be consistent